broker/alert_acquisition/ztf.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+# -*- coding: UTF-8 -*-
+
+"""Retrieve and parse alerts from the ZTF."""
+
+import os
+
+import pandas as pd
+
+from .._utils import setup_log
+from ..ztf_archive import iter_alerts
+
+if 'RTD_BUILD' not in os.environ:
+    from google.cloud import error_reporting
+
+    error_client = error_reporting.Client()
+    log = setup_log('ztf_acquisition')
+
+# Temporary stop gap until the live alert stream is accessible
+alert_iterable = None
+from warnings import warn
+
+warn('This module is currently in progress and relies on the ZTF Public Alerts'
+     'Archive, not the live ZTF stream.')
+
+
+def get_alerts(num_alert):
+    """Fetch alerts from the ZTF alert stream
+
+    Todo:
+        Function currently returns 10 alerts from the ZTF archive module.
+        Get data from the alert stream instead of the ZTF Archive.
```

Comment:
"Get data from the ZTF Alert Stream instead of the ZTF Archive."

To help clarify that both are ZTF, it's just the stream vs. the archive.